src/transport/RendezvousSession.cpp

```diff
@@ -40,32 +41,29 @@ namespace chip {
 
 CHIP_ERROR RendezvousSession::Init(const RendezvousParameters & params)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
-
     mParams = params;
-    VerifyOrExit(mDelegate != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
-    VerifyOrExit(mParams.HasSetupPINCode(), err = CHIP_ERROR_INVALID_ARGUMENT);
+    ReturnErrorCodeIf(mDelegate == nullptr, CHIP_ERROR_INCORRECT_STATE);
+    ReturnErrorCodeIf(!mParams.HasSetupPINCode(), CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Why that and not `VerifyOrReturnError` here?  It's not clear to me whether there's a principled way to decide between these two macros...

Reply:
I have no clear way to differentiate between the two, I changed this as an example.

I believe both are readable. My best guess of choosing would be 'negations are hard to follow' so the correct way here seems indeed VerifyOrReturnError because HasSetupPinCode is otherwise negated. I will update the code.